basin_setup/CoreConfig.ini

```diff
@@ -0,0 +1,77 @@
+[generate_topo]
+
+basin_shapefile:
+type = CriticalFilename,
+description = Path to shapefile that defines the basin in UTM projection
+
+cell_size:
+type = float,
+default = 50.0,
+description = Pixel size to use for the basin in meters
+
+dem_file:
+type = CriticalFilename,
+description = Geotiff digital elevation file
+
+vegetation_folder:
+type = Directory,
+description = Location to check for veg data or download vegetation data files to
+
+vegetation_dataset:
+type = string,
+default = landfire_1.4.0,
+options = [landfire_1.4.0 landfire_2.1.0 None],
+description = Type of vegetation dataset to use. The Landfire numbers refer to their version number.
+
+output_folder:
+type = Directory,
+default = ./generate_topo,
+description = Folder for output data from generate_topo
+
+pad_domain:
+type = int list,
+default = [0 0 0 0],
+description = Number of cells to add on to the domain for each side. Format is [Left Bottom Rightm Top]
+
+coordinate_extent:
+type = float list,
+description = Coordinates to set the extents of the final topo to.
+              Format is [Left Bottom Right Top]
+
+leave_intermediate_files:
+type = bool,
+default = False,
+description = Debug flag prints out more info and does not delete the working folder
+
+sub_basin_files:
+type = filename list,
+description = provide a file list of subassin shapefiles you want to be added as masks
+
+basin_name:
+type = string,
+default = Full Basin,
+description = Long name for the basin total mask
+
+veg_params_csv:
+type = DiscretionaryCriticalFilename,
+description = Provide a csv defining veg tau and veg k values for the available vegetation
+              classes. Any veg classes found in the topo not listed in the csv will throw
+              an error
```

Comment:
Throw an error? Or just a warning?

Reply:
I don't understand. So if someone brings their own csv with tau and k corresponding to the veg classes?

Reply:
Yup. This would be a way to either override our decisions or provide values for classes that we've missed (like the Feather).